PopupBridge/src/main/java/com/braintreepayments/api/PopupBridgeClient.java

```diff
@@ -17,8 +17,12 @@
 
 public class PopupBridgeClient {
 
-    public static final String POPUP_BRIDGE_NAME = "popupBridge";
-    public static final String POPUP_BRIDGE_URL_HOST = "popupbridgev1";
+    // TODO: consider using a `com.braintreepayments...` prefixed request key
```

Comment:
General question, is this TODO for this PR or for a future task?

Reply:
Yeah I had `NEXT_MAJOR_VERSION` initially but it may be possible with out a major version bump. I replaced with `TODO: consider` but to be real it probably won't happen until Android `v5`.

Reply:
Ended up reverting the next major version message.